en/firefox/browsers/chromebook.ftl

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the { -brand-name-mozilla } Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/browsers/chromebook/
+
+# HTML page title
+
+browsers-chromebook-get-firefox-browser = Get { -brand-name-firefox-browser } for { -brand-name-chromebook }
+
+browsers-chromebook-so-youve-got = So you’ve got a { -brand-name-chromebook } — now it’s time to get the independent, automatically private { -brand-name-firefox-browser } to protect your personal data.
+
+browsers-chromebook-browsers-chromebook-get-firefox-for = Get { -brand-name-firefox } for { -brand-name-android }
+
+browsers-chromebook-get-firefox-desktop = Get { -brand-name-firefox } Desktop for { -brand-name-chromebook }
+
+browsers-chromebook-x86-based-chromebook = *x86 based { -brand-name-chromebook } running { -brand-name-chromeos } 80 or later
+
+browsers-chromebook-dropdown-copy = Get { -brand-name-firefox } for { -brand-name-chromebook }
+
+# The span here is to display an underline underneath "Firefox"
+browsers-chromebook-why-get-firefox = Why get <span>{ -brand-name-firefox }</span> for { -brand-name-chromebook }?
+
+browsers-chromebook-while-a-chromebook = While a { -brand-name-chromebook } already has { -brand-name-chrome } installed, downloading and using { -brand-name-firefox } as your go-to browser provides you with a few benefits:
+
+browsers-chromebook-alwayson-tracking-protection = Always-on tracking protection: by default, { -brand-name-firefox } runs Enhanced Tracking Protection (ETP) to protect your personal data from known ad trackers, social trackers and cryptomining scripts that follow you around the web.
+
+browsers-chromebook-supporting-independent-technology = Supporting independent technology: since most major browsers are now running on { -brand-name-chromium }, there are fewer options that don’t feed into the Big Tech machine. { -brand-name-firefox } is backed by a non-profit and is dedicated to fixing the internet. Using { -brand-name-firefox } makes you a part of that.
+
+browsers-chromebook-is-it-hard = Is it hard to install { -brand-name-firefox } for { -brand-name-chromebook }?
+
+browsers-chromebook-we-wish-it = We wish it were more straightforward, but your { -brand-name-chromebook } would prefer to keep you in the { -brand-name-google-play } ecosystem. But, we think it’s worth the effort to install the { -brand-name-firefox } browser to your { -brand-name-chromebook } – and we’ve got resources to help you if you need them. There are two ways to get { -brand-name-firefox } on your device.
```

Comment:
There are two `but` back to back in this sentence. Should we change the second one to `However, we think...`?